src/test/java/temp/tests/TestOpenPageChrome.java

```diff
@@ -1,25 +1,27 @@
-package tests;
+package temp.tests;
 
-import engine.drivers.WebDriverFactory;
 import enums.BrowserName;
 import org.junit.Assert;
 import org.junit.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.LoginPage;
 
 import java.time.Duration;
 
+import static engine.drivers.WebDriverFactory.*;
 import static engine.property.manager.PropertyManager.PropertyKeys.BASE_URL;
-import static engine.property.manager.PropertyManager.getPropertyManagerInstance;
 import static engine.property.manager.PropertyManager.getProperty;
+import static engine.property.manager.PropertyManager.getPropertyManagerInstance;
 
 public class TestOpenPageChrome {
     @Test
     public void openPage() {
         getPropertyManagerInstance();
-        WebDriver driver = WebDriverFactory.getWebDriver(BrowserName.CHROME);
+        WebDriver driver = getWebDriverInstance(BrowserName.CHROME);
```

Comment:
I would move web driver init to the basePage class or to some base test class - I think there was a possibility to do that so TestOpenPageChrome would extend some baseTest class which can do the initialization:)

Reply:
@Fishgame1 Done - web driver initialization is now being done only in BasePage

Reply:
Ok, so we can remove that line here right?:)

Reply:
@Fishgame1 Web driver init left only in BasePage class